todoCLI/todoCLI.js

```diff
@@ -0,0 +1,111 @@
+const fs = require("fs");
+const readline = require("readline");
+
+let items = [];
+
+const r1 = readline.createInterface( {
+    input: process.stdin,
+    output: process.stdout
+});
+
+function startTodoCLI() {
+
+    r1.question('\nWelcome to Todo CLI! \n -------------------- \n(v) View • (n) New • (cX) Complete • (dX) Delete • (q) Quit \n>', 
+    (option) => {
+
+        //console.log('Option entered is:', option);
+
+        if(option !== 'q') {
```

Comment:
Nested if is not needed here. Checking for quit first will have the same result.

Ideally a switch statement is used here for readability.